hyperspy/tests/mva/test_decomposition.py

```diff
@@ -229,3 +253,38 @@ def test_decomposition_supported_return_false(self):
                 algorithm=algorithm,
                 return_info=False,
                 output_dimension=1) is None
+
+
+@pytest.mark.skipif("sys.platform == 'darwin'")
+class TestPlotExplainedVarianceRatio:
+
+    def setup_method(self, method):
+        MY_PATH = os.path.dirname(__file__)
```

Comment:
It would be better to generate a dataset for this (even if just with a seeded ``np.random``) because otherwise, if the FEI readers gets broken, it'll seem that this plotting function is broken too.